modules-legacy/viewport-mercator-project/package.json

```diff
@@ -18,13 +18,9 @@
     "type": "git",
     "url": "https://github.com/uber-web/math.gl.git"
   },
-  "files": [
-    "index.js",
-    "module.js"
-  ],
   "main": "index.js",
   "module": "module.js",
   "dependencies": {
```

Comment:
Can't do a suggestion on delete lines, but I would do.
```js
  "files": [	
    "index.d.ts",
    "index.js",	
    "module.js"	
  ],	
  "types": "index.d.ts",
  "main": "index.js",
  "module": "module.js",
```

Reply:
I do not think "files" is still necessary as all files in this directory need to be published. I will add the "types" entry.